### PR TITLE
Add release announcement of add-ons 1.9 and plug-in API 1.6

### PR DESCRIPTION
**Pull request by tkrille**

### CODE REVIEW

**Comment by dacrome** [comment]
I think it's mergable. What do you think @wallner?

**Comment by tkrille** [comment]
I've made this a published post and reworded some phrases like @wallner did in 4bec88a.

File: _posts/2016-08-06-release-add-ons-1.9.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title:  "Release: Add-Ons 1.9"
+---
+
+We are proud to announce the availability of Add-On Administration 1.9, Add-On Self Administration 1.9,
+and Add-On Self Administration Plug-In API 1.6.
+These releases are mainly fixing bugs, but there's also a new feature in the Add-On Administration.
```
Comment (wallner) [inline _posts/2016-08-06-release-add-ons-1.9.md]: Maybe better: "These releases are mainly fixing bugs. but there is one new feature in the Add-On Administration" ?

File: _posts/2016-08-06-release-add-ons-1.9.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+This release officially deprecates the support for OSIAM 2.x, which will be removed in version 1.12 or 2.0.
+Furthermore, this will probably be the last release of all add-ons, as they will be merged into the core server soon.
```
Comment (wallner) [inline _posts/2016-08-06-release-add-ons-1.9.md]: "Also, this is probably the last stand-alone release of all add-ons"?

File: _posts/2016-08-06-release-add-ons-1.9.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+The artifacts have been deployed on Bintray and in case of the Add-On Self Administration API on the Maven central.
+The coordinates of the plug-in API are `org.osiam:addon-self-administration-plugin-api:1.6:jar`.
+You can download the files using one of the following links:
```
Comment (wallner) [inline _posts/2016-08-06-release-add-ons-1.9.md]: "The files are available using one of the following links:"

File: _posts/2016-08-06-release-add-ons-1.9.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+[Add-On Administration 1.9](https://dl.bintray.com/osiam/downloads/addon-administration/1.9/)
+
+[Add-On Self Administration 1.9](https://dl.bintray.com/osiam/downloads/addon-self-administration/1.9/)
+
+[Add-On Self Administration Plug-In API 1.6](https://dl.bintray.com/osiam/OSIAM/org/osiam/addon-self-administration-plugin-api/1.6/)
+
+The following is an overview of the most important changes.
+See the full changelog of the [Add-On Administration](https://github.com/osiam/addon-administration/blob/1.9/CHANGELOG.md),
```
Comment (wallner) [inline _posts/2016-08-06-release-add-ons-1.9.md]: The full changelog is available here:

File: _posts/2016-08-06-release-add-ons-1.9.md (hunk 2 above)
Comment (tkrille) [inline _posts/2016-08-06-release-add-ons-1.9.md]: There will be also no non-standalone release. What I mean here, is that there will be no further releases whatsoever. Functionality will be re-implemented in the core.

File: _posts/2016-08-06-release-add-ons-1.9.md (hunk 4 above)
Comment (tkrille) [inline _posts/2016-08-06-release-add-ons-1.9.md]: I rather use the plural form and make the links a list.

**Comment by tkrille** [comment]
Thanks for the comments @wallner. I've update my PR.

File: _posts/2016-08-06-release-add-ons-1.9.md (hunk 1 above)
Comment (tkrille) [inline _posts/2016-08-06-release-add-ons-1.9.md]: Yes

File: _posts/2016-08-06-release-add-ons-1.9.md (hunk 3 above)
Comment (tkrille) [inline _posts/2016-08-06-release-add-ons-1.9.md]: Yes

File: _posts/2016-08-06-release-add-ons-1.9.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+---
+title:  "Release: Add-Ons 1.9"
+---
+
+We are proud to announce the availability of Add-On Administration 1.9, Add-On Self Administration 1.9,
+and Add-On Self Administration Plug-In API 1.6.
+These releases are mainly fixing bugs.
+But there is one new feature in the Add-On Administration.
+This release officially deprecates the support for OSIAM 2.x, which will be removed in version 1.12 or 2.0.
+Furthermore, this will probably be the last release of all add-ons, as they will be merged into the core server soon.
```
Comment (wallner) [inline _posts/2016-08-06-release-add-ons-1.9.md]: Can we please replace furthermore?

Comment (tkrille) [inline _posts/2016-08-06-release-add-ons-1.9.md]: With what?

Comment (tkrille) [inline _posts/2016-08-06-release-add-ons-1.9.md]: What about "moreover"?

Comment (wallner) [inline _posts/2016-08-06-release-add-ons-1.9.md]: What about "Additionally"?

Comment (tkrille) [inline _posts/2016-08-06-release-add-ons-1.9.md]: What about using no adverb at all?

**Comment by wallner** [comment]
Thank you!

**Pull request merged**